pkg/cluster/storage.go

```diff
@@ -51,6 +34,9 @@ func (c *Cluster) save(ctx context.Context) error {
 		if err.Error() == "key exists" {
 			logrus.Warnln("Bootstrap key exists. Please follow documentation updating a node after restore.")
 			return nil
+		} else if strings.Contains(err.Error(), "not supported for learner") {
+			logrus.Infof("this server is a learner. Skipping saving bootstrap data")
```

Comment:
This isn't something the user needs to concern themselves with, lets just debug it.
```suggestion
			logrus.Debug("Skipping bootstrap data save on learner.")
```